doobie/src/main/scala/za/co/absa/db/fadb/doobie/postgres/circe/implicits/package.scala

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2022 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.db.fadb.doobie.postgres.circe
+
+import cats.Show
+import cats.data.NonEmptyList
+import doobie.{Get, Put}
+import io.circe.Json
+import org.postgresql.jdbc.PgArray
+import org.postgresql.util.PGobject
+import io.circe.parser._
+
+import scala.util.{Failure, Success, Try}
+
+package object implicits {
+
+  private implicit val showPgArray: Show[PgArray] = Show.fromToString
+
+  implicit val jsonArrayPut: Put[List[Json]] = {
+    Put.Advanced
+      .other[PGobject](
+        NonEmptyList.of("json[]")
+      )
+      .tcontramap { a =>
+        val o = new PGobject
+        o.setType("json[]")
+        o.setValue(jsonListToPGJsonArrayString(a))
+        o
+      }
+  }
+
+  implicit val jsonArrayGet: Get[List[Json]] = {
+    Get.Advanced
+      .other[PgArray](
+        NonEmptyList.of("json[]")
+      )
+      .temap(pgArray => pgArrayToListOfJson(pgArray))
+  }
+
+  implicit val jsonbArrayPut: Put[List[Json]] = {
```

Comment:
what about JSONB get utils?

Reply:
It seems to work for both :D

Reply:
But I am not against having both defined. Actually let me push it there.

Reply:
really?

because the GET implements `json[]`:

```scala
  implicit val jsonArrayGet: Get[List[Json]] = {
    Get.Advanced
      .other[PgArray](
        NonEmptyList.of("json[]")
      )
      .temap(pgArray => pgArrayToListOfJson(pgArray))
  }
```

perhaps for the sake of consistency in the naming we should either: implement the second one with `jsonb[]` or name this one somehow so that it's understandable that it's also for JSONB GET?

Reply:
discussed on a call

Reply:
implemented like this after discussion on a call
```scala
implicit val jsonOrJsonbArrayGet: Get[List[Json]] = {
    Get.Advanced
      .other[PgArray](
        NonEmptyList.of("json[]")
      )
      .temap(pgArray => pgArrayToListOfJson(pgArray))
  }
```

Reply:
Somwhat longer but acceptable, IMHO.